cohttp-lwt/src/s.ml

```diff
@@ -37,6 +37,27 @@ end
 module type Client = sig
   type ctx
 
+  module Connection : sig
+    type t
+
+    val connect :
+      ?finalise:(t -> unit Lwt.t) ->
+      ?ctx:ctx ->
+      Uri.t -> (* XXX this should actually be something like Conduit.endp *)
+      t
+
+    val close : t -> unit Lwt.t
+    val is_closed : t -> bool
+    val notify : t -> unit Lwt.t
+    val length : t -> int
+
+    val request :
+      t ->
+      ?body:Body.t ->
+      Cohttp.Request.t ->
+      (Cohttp.Response.t * Body.t) Lwt.t
+  end
+
   module Conn_cache : sig
```

Comment:
```suggestion
  module Connection_cache : sig
```
A few more characters do not harm.
The module with the signature will need to be documented before this can be merged